exercises/phone-number/README.md

```diff
@@ -0,0 +1,18 @@
+# Basic Reason Template
+
+Hello! This project allows you to quickly get started with Reason and BuckleScript. If you wanted a more sophisticated version, try the `react` template (`bsb -theme react -init .`).
+
```

Comment:
please add the problem statement to this readme. https://github.com/exercism/problem-specifications/blob/master/exercises/phone-number/description.md

Reply:
@tejasbubane yep, it still being a _work in progress_, it is not ready to be merged or reviewed but thanks anyway.